components/proto-derive/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This crate provides macros for deriving some useful methods and traits for the exonum services.
+
+#![recursion_limit = "128"]
+#![deny(unsafe_code, bare_trait_objects)]
+#![warn(missing_docs, missing_debug_implementations)]
+
+extern crate proc_macro;
+
+mod pb_convert;
+
+use darling::FromMeta;
+use proc_macro::TokenStream;
+use quote::ToTokens;
+use syn::{Attribute, NestedMeta};
+
+/// ProtobufConvert attribute macros.
```

Comment:
Nit: why plural (macros)? Also, an example or two would be great, especially for `enum`s.